core/src/main/resources/org/fao/geonet/api/Messages_fre.properties

```diff
@@ -104,3 +104,6 @@ self_registration_disabled=User self-registration is disabled
 recaptcha_not_valid=Recaptcha is not valid
 metadata.title.createdFromTemplate=Copie du mod\u00e8le %s cr\u00E9\u00E9e le %s
 metadata.title.createdFromRecord=Copie de la fiche %s cr\u00E9\u00E9e le %s
+username.field.required=Username is required
+password.field.length=Password size should be between {min} and {max} characters
+password.field.invalid=Password must contain at least 1 uppercase, 1 lowercase, 1 number and 1 symbol. Symbols include: `~!@#$%^&*()-_=+[]{}\\|;:'",.<>/?');
```

Comment:
Would it be possible to add an additional setting so that the catalog admin can change the message string? At present, a catalog admin could change the regex pattern, but would then need someone with access to the code to change the message